src/js/mainFunctions.js

```diff
@@ -3,8 +3,28 @@ import ScoreBoard from './ScoreBoard.js';
 const scoreBoard = new ScoreBoard();
 
 const form = document.getElementById('form');
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
-  scoreBoard.addNewScore();
-  scoreBoard.displayScor();
-});
\ No newline at end of file
+const formAction = () => {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const name = document.getElementById('name');
+    const score = document.getElementById('score');
+
+    if (name.value !== '' && score.value !== '') {
+      scoreBoard.addScore(name.value, score.value);
+      name.value = '';
+      score.value = '';
+    }
+    scoreBoard.displayScor();
+  });
+};
+const refresh = document.querySelector('.refresh');
+const refreshAction = () => {
+  // console.log('hhhhhhhhhhhh');
```

Comment:
As mentioned above Javascript best practises, we do not commit console.log to our repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at our website.so please remove it.